src/commands.js

```diff
@@ -86,6 +87,7 @@ export function addColumn(tr, {map, tableStart, table}, col) {
       );
     }
   }
+  typeInheritance(tr, tableStart);
```

Comment:
This would be needed in the future if either (1) we changed text cells so they needed rendering or (2) we allowed adding columns whose type starts out as non-text. But you are right that it does nothing for now, since new column cells don't need rendering.

Reply:
ok Im fine with leaving this as is or adding TODO

Reply:
I'll change it to TODO